bulk/sensor_bulk_load-AssetRecord.csv

```diff
@@ -1273,6 +1274,7 @@ CGINS-FDCHPA-70601,,Sensor,0,Direct Covariance Flux: FDCHP Series A,ANEMOMETER W
 CGINS-FDCHPA-71403,,Sensor,0,Direct Covariance Flux: FDCHP Series A,ANEMOMETER WINDMASTER PRO,Gill,WINDMASTER PRO,171403,,20170505,,,Woods Hole Oceanographic Institution
 CGINS-FDCHPA-74102,,Sensor,0,Direct Covariance Flux: FDCHP Series A,ANEMOMETER WINDMASTER PRO,Gill,WINDMASTER PRO,174102,,20180109,,,Woods Hole Oceanographic Institution
 CGINS-FDCHPA-75008,,Sensor,0,Direct Covariance Flux: FDCHP Series A,ANEMOMETER WINDMASTER PRO,Gill,WINDMASTER PRO,175008,,,,,Woods Hole Oceanographic Institution
+CGINS-FDCHPB-03902,,Sensor,0,Direct Covariance Flux: FDCHP Series B,ANEMOMETER WINDMASTER PRO,Gill,WINDMASTER PRO R3-50,203902,,,,,Woods Hole Oceanographic Institution

```

Comment:
Needs to be changed from:
FDCHP Series B,ANEMOMETER WINDMASTER PRO,Gill,WINDMASTER PRO R3-50,203902
to 
FDCHP Series B,ANEMOMETER R3-50,Gill,R3-50,203902

The WindMaster Pro and R3-50 are different anemometers made by Gill.

Reply:
corrected.